### PR TITLE
Electron prod

### PR DESCRIPTION
**Pull request by dwalintukan**
Mac and Windows tested to build fine. Need to use a Linux based machine to build Linux release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+4. `npm start`
+
+5. App at `127.0.0.1:5555` or GraphiQL at `127.0.0.1:5555/graphiql`
 
-5. start bodhi-graphql at 5555
+# Release
+We are currently using [Electron](https://github.com/electron/electron) and [Electron-Builder](https://github.com/electron-userland/electron-builder) to package our apps into native format depending on OS.
 
-`npm start`
+1. `npm run insall` to install all node modules from npm and yarn
```
Comment (frankobe) [inline README.md]: typo `npm run install`

Comment (frankobe) [reply README.md]: why not include `npm install` in `npm run build:xxx`

File: package.json [hunk 2]
```diff
@@ -1,27 +1,106 @@
 {
-  "name": "bodhi-app",
-  "version": "0.2.0",
-  "description": "An open source blockchain explorer for monitoring Bodhi platform",
+  "name": "BodhiPM",
```
Comment (frankobe) [inline package.json]: Bodhi-App? PM for 'Prediction Market' isnt comprehensive

File: package.json [hunk 3]
```diff
@@ -1,27 +1,106 @@
...
     "qtum",
     "bodhi",
     "api",
     "graphql"
   ],
-  "author": "bodhi.network",
-  "license": "ISC",
-  "bin": "src/index.js",
+  "scripts": {
+    "insall": "npm install && ./node_modules/.bin/yarn install",
```
Comment (frankobe) [inline package.json]: can we use `install` so all dependencies can be installed by `npm run:install`. if not, how abt `install-dep`

Comment (dwalintukan) [reply package.json]: Tried using `install` as the name but that didn't work. I'll use `install-dep`

File: package.json (hunk 2 above)
Comment (dwalintukan) [reply package.json]: How about `BodhiPredictionMarket`? Just think there's a possibility of maybe having a different app in the future.

File: README.md (hunk 1 above)
Comment (dwalintukan) [reply README.md]: changed to `npm run install-dep`

File: package.json (hunk 2 above)
Comment (frankobe) [reply package.json]: Just check with TT, let use`Bodhi`

Comment (dwalintukan) [reply package.json]: 👍

**Pull request merged**